os-config.tpl.yml

```diff
@@ -161,6 +161,19 @@ rancher:
       read_only: true
       volumes:
       - /var/lib/docker:/var/lib/docker
+    logrotate:
+      image: {{.OS_REPO}}/os-logrotate:{{.VERSION}}{{.SUFFIX}}
+      command: /usr/sbin/logrotate -v /etc/logrotate.conf
+      labels:
+        io.rancher.os.createonly: "true"
+        io.rancher.os.scope: system
+        io.rancher.os.before: system-cron
```

Comment:
It gets them, it just doesn't log the event. Initially I thought it would be too verbose, but it is causing more confusion then anything. There is an open issue to add logging.